### DISCUSSION TITLE
How do I disable authentication?

### DISCUSSION THREAD

**Discussion by HookedBehemoth** [bluenviron/mediamtx#3719] [Questions]
### Question

I can't seem to figure out how to disable authentication on the newer version. What magic spell do I need to cast for this to work like it did on 1.5.1?

**Comment by aler9** [comment]
Regarding publishing and reading streams, there are no authentication differences between v1.5.1 and v1.11.0.
Both allows to publish and read streams with no authentication.
Nothing more can be asserted since you didn't provide anything that allows to replicate the issue (configuration files, server logs). If you still have problems, open a bug report.
